src/services/schemaCompareService.ts

```diff
@@ -10,11 +10,162 @@ import * as mssql from "vscode-mssql";
 export class SchemaCompareService implements mssql.ISchemaCompareService {
     constructor(private _client: SqlToolsServiceClient) {}
 
+    public schemaCompare(
+        operationId: string,
+        sourceEndpointInfo: mssql.SchemaCompareEndpointInfo,
+        targetEndpointInfo: mssql.SchemaCompareEndpointInfo,
+        taskExecutionMode: mssql.TaskExecutionMode,
+        deploymentOptions: mssql.DeploymentOptions,
+    ): Thenable<mssql.SchemaCompareResult> {
+        const params: mssql.SchemaCompareParams = {
+            operationId: operationId,
+            sourceEndpointInfo: sourceEndpointInfo,
+            targetEndpointInfo: targetEndpointInfo,
+            taskExecutionMode: taskExecutionMode,
+            deploymentOptions: deploymentOptions,
+        };
+
+        return this._client.sendRequest(
+            schemaCompareContracts.SchemaCompareRequest.type,
+            params,
+        );
+    }
+
+    public schemaCompareGenerateScript(
```

Comment:
I think we can improve upon ADS here by removing the "schemaCompare" from "schemaCompareGenerateScript" and all the others that are more verbose like this.  It's already in the SchemaCompareService, so that should be obvious enough.